app/assets/javascripts/site.js

```diff
@@ -237,14 +237,17 @@ var Downloads = {
 
   getOSFilter: function() {
     var os = location.href.substring(location.href.lastIndexOf("/") + 1);
-    return os === 'linux' || os === 'mac' || os === 'windows'
-      ? os
-      : '';
+    return os === 'linux' || os === 'mac' || os === 'windows' || os === 'android' || os === 'ios' ? os : '';
+  },
+
+  capitalizeOS: function(os) {
+    const platforms = {"linux": "Linux", "mac": "Mac", "windows": "Windows", "android": "Android", "ios": "iOS"};
+    return platforms[os];
```

Comment:
I wanted to reuse the gui_helper.rb dictionary, but we would need to move the .js file into a .js.erb file or something, from what I've read: https://stackoverflow.com/a/20590659/4398050